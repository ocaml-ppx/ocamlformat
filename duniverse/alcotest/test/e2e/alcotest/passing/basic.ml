let () =
  let open Alcotest in
  let id () = () in
  run __FILE__
    [
      ( "test-a",
        [
          test_case "First test case" `Quick id;
          test_case "Second test case" `Quick id;
        ] );
      ("test-b", [ test_case "Third test case" `Quick id ]);
      ("test-c", [ test_case "Fourth test case" `Slow id ]);
    ]
