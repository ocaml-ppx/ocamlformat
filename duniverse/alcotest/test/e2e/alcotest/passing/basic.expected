Testing `test/e2e/alcotest/passing/basic.ml'.
This run has ID `<uuid>'.

  [OK]          test-a          0   First test case.
  [OK]          test-a          1   Second test case.
  [OK]          test-b          0   Third test case.
  [OK]          test-c          0   Fourth test case.

Full test results in `<build-context>/_build/_tests/<test-dir>'.
Test Successful in <test-duration>s. 4 tests run.
